### PR TITLE
Added constants on manager/controllers/ dir

### PR DESCRIPTION
**Pull request by Raviv-S**
Signed-off-by: Raviv Schaffer <raviv.schaffer@ibm.com>

### CODE REVIEW

File: manager/controllers/app/blueprint_controller.go [hunk 1]
```diff
@@ -40,6 +40,10 @@ import (
 	"fybrik.io/fybrik/pkg/logging"
 )
 
+const (
+	blueprintLiteral = "blueprint"
```
Comment (shlomitk1) [inline manager/controllers/app/blueprint_controller.go:44]: BlueprintKind

File: manager/controllers/app/fybrikapplication_controller.go [hunk 2]
```diff
@@ -69,16 +69,17 @@ type ApplicationContext struct {
 }
 
 const (
-	ApplicationTaxonomy = "/tmp/taxonomy/fybrik_application.json"
-	DataCatalogTaxonomy = "/tmp/taxonomy/datacatalog.json#/definitions/GetAssetResponse"
+	ApplicationTaxonomy      = "/tmp/taxonomy/fybrik_application.json"
+	DataCatalogTaxonomy      = "/tmp/taxonomy/datacatalog.json#/definitions/GetAssetResponse"
+	fybrikapplicationLiteral = "fybrikapplication"
```
Comment (shlomitk1) [inline manager/controllers/app/fybrikapplication_controller.go:74]: FybrikApplicationKind

File: manager/controllers/app/fybrikmodule_controller.go [hunk 3]
```diff
@@ -35,11 +35,12 @@ type FybrikModuleReconciler struct {
 const (
 	ModuleTaxonomy                 = "/tmp/taxonomy/fybrik_module.json"
 	ModuleValidationConditionIndex = 0
+	fybrikModuleLiteral            = "FybrikModule"
```
Comment (shlomitk1) [inline manager/controllers/app/fybrikmodule_controller.go:38]: FybrikModuleKind

File: manager/controllers/app/plotter_controller.go [hunk 4]
```diff
@@ -30,6 +30,12 @@ import (
 	"fybrik.io/fybrik/pkg/multicluster"
 )
 
+const (
+	plotterLiteral           = "Plotter"
```
Comment (shlomitk1) [inline manager/controllers/app/plotter_controller.go:34]: PlotterKind

File: manager/controllers/app/plotter_controller.go [hunk 5]
```diff
@@ -30,6 +30,12 @@ import (
...
+	noRemoteBlueprintWarnMsg = "Could not yet find remote blueprint"
+	plotterReadyMsg          = "Plotter is ready!"
```
Comment (shlomitk1) [inline manager/controllers/app/plotter_controller.go:36]: I don't think messages should be defined as constants.

File: manager/controllers/app/plotter_generator.go [hunk 6]
```diff
@@ -30,6 +30,7 @@ import (
 const (
 	objectKeyHashLength  = 10
 	bucketNameHashLength = 10
+	s3Literal            = "s3"
```
Comment (shlomitk1) [inline manager/controllers/app/plotter_generator.go:33]: Should be S3, and I think we already have this constant defined somewhere

File: manager/controllers/mockup/clusterlister.go [hunk 7]
```diff
@@ -7,6 +7,10 @@ import (
 	"fybrik.io/fybrik/pkg/multicluster"
 )
 
+const (
+	kubernetesLiteral = "kubernetes"
```
Comment (shlomitk1) [inline manager/controllers/mockup/clusterlister.go:11]: VaultAuthPath

File: manager/controllers/mockup/datacatalog.go [hunk 8]
```diff
@@ -19,6 +19,11 @@ import (
 const (
 	dummyResourceName = "xxx"
 	dummyCredentials  = "dummy"
+	kafkaDeserializer = "io.confluent.kafka.serializers.json.KafkaJsonSchemaDeserializer"
+	s3Literal         = "s3"
+	db2Literal        = "db2"
+	kafkaLiteral      = "kafka"
```
Comment (shlomitk1) [inline manager/controllers/mockup/datacatalog.go:25]: S3, DB2, Kafka - although I think we have their definition elsewhere
All the rest should not be constants. Maximum, reuse values using a string variable in the code.

File: manager/controllers/mockup/policymanager.go [hunk 9]
```diff
@@ -16,6 +16,13 @@ import (
 	"fybrik.io/fybrik/pkg/random"
 )
 
+const (
+	denyKey         = "Deny"
```
Comment (shlomitk1) [inline manager/controllers/mockup/policymanager.go:20]: DenyAction

File: manager/controllers/mockup/policymanager.go [hunk 10]
```diff
@@ -16,6 +16,13 @@ import (
...
+	nameKey         = "name"
+	theshireLiteral = "theshire"
+	redactActionKey = "RedactAction"
```
Comment (shlomitk1) [inline manager/controllers/mockup/policymanager.go:23]: RedactAction

File: manager/controllers/mockup/policymanager.go [hunk 11]
```diff
@@ -16,6 +16,13 @@ import (
 	"fybrik.io/fybrik/pkg/random"
 )
 
+const (
+	denyKey         = "Deny"
+	nameKey         = "name"
+	theshireLiteral = "theshire"
```
Comment (shlomitk1) [inline manager/controllers/mockup/policymanager.go:22]: no need for constant, just reuse a string variable

File: manager/controllers/utils/configuration.go [hunk 12]
```diff
@@ -25,6 +25,7 @@ const (
 	PrettyLoggingKey                  string = "PRETTY_LOGGING"
 	CatalogProviderNameKey            string = "CATALOG_PROVIDER_NAME"
 	DatapathLimitKey                  string = "DATAPATH_LIMIT"
+	envVarLiteral                     string = "Env variable "
```
Comment (shlomitk1) [inline manager/controllers/utils/configuration.go:28]: Not a constant, you can remove it from the message.

File: manager/controllers/app/plotter_controller.go (hunk 5 above)
Comment (Raviv-S) [reply manager/controllers/app/plotter_controller.go:36]: Should I change them to vars instead or to not save them at all?

Comment (shlomitk1) [reply manager/controllers/app/plotter_controller.go:36]: Not save them at all. Inside the code assign a string variable and reuse it for both conditions and log messages.

File: manager/controllers/utils/configuration.go (hunk 12 above)
Comment (Raviv-S) [reply manager/controllers/utils/configuration.go:28]: Do you mean to add lint-ignore? Or just change it to vars?

File: manager/controllers/app/plotter_controller.go [hunk 13]
```diff
@@ -327,6 +330,7 @@ func (r *PlotterReconciler) reconcile(plotter *app.Plotter) (ctrl.Result, []erro
 	blueprintsMap := r.getBlueprintsMap(plotter)
 
 	var errorCollection []error
+	var noRemoteBlueprintWarnMsg = "Could not yet find remote blueprint"
```
Comment (shlomitk1) [inline manager/controllers/app/plotter_controller.go:333]: noRemoteBlueprintWarnMsg := "Could ..."

File: manager/controllers/app/plotter_controller.go [hunk 14]
```diff
@@ -459,15 +463,15 @@ func (r *PlotterReconciler) reconcile(plotter *app.Plotter) (ctrl.Result, []erro
 	plotter.Status.ObservedGeneration = plotter.ObjectMeta.Generation
 	plotter.Status.ObservedState.Ready = isReady
 	plotter.Status.Assets = assetToStatusMap
-
+	var plotterReadyMsg = "Plotter is ready!"
```
Comment (shlomitk1) [inline manager/controllers/app/plotter_controller.go:466]: plotterReadyMsg := "Plotter...

File: manager/controllers/mockup/datacatalog.go [hunk 15]
```diff
@@ -63,19 +58,23 @@ func NewTestCatalog() *DataCatalogDummy {
...
 	geoExternal := "neverland"
 
 	// TODO(roee88): some of these are also defined in ifdetails.go
 	var csvFormat taxonomy.DataFormat = "csv"
 	var parquetFormat taxonomy.DataFormat = "parquet"
 	var jsonFormat taxonomy.DataFormat = "json"
+	var dummyResourceName = "xxx"
+	var dummyCredentials = "dummy"
+	var kafkaDeserializer = "io.confluent.kafka.serializers.json.KafkaJsonSchemaDeserializer"
+	var sslTruststore = "xyz123"
```
Comment (shlomitk1) [inline manager/controllers/mockup/datacatalog.go:71]: Please keep the same style, i.e. <var_name> := <value> without "var" declaration

File: manager/controllers/mockup/policymanager.go [hunk 16]
```diff
@@ -40,6 +45,8 @@ func (m *MockPolicyManager) GetPoliciesDecisions(input *policymanager.GetPolicyD
 	log.Printf("Received OpenAPI request in mockup GetPoliciesDecisions: ")
 	log.Printf("ProcessingGeography: %s", input.Action.ProcessingLocation)
 	log.Printf("Destination: " + input.Action.Destination)
+	var nameKey = "name"
+	var theshireLiteral = "theshire"
```
Comment (shlomitk1) [inline manager/controllers/mockup/policymanager.go:49]: same comment

File: manager/controllers/utils/configuration.go [hunk 17]
```diff
@@ -93,11 +93,12 @@ func DefaultTestConfiguration(t ginkgo.GinkgoTInterface) {
 }
 
 func logEnvVariable(log zerolog.Logger, key string) {
+	var envVarLiteral = "Env variable "
 	value, found := os.LookupEnv(key)
 	if found {
-		log.Info().Msg("Env variable " + key + " set to \"" + value + "\"")
+		log.Info().Msg(envVarLiteral + key + " set to \"" + value + "\"")
```
Comment (shlomitk1) [inline manager/controllers/utils/configuration.go:99]: log.Info().Msgf("%s set to %s",key,value)

File: manager/controllers/utils/configuration.go [hunk 18]
```diff
@@ -93,11 +93,12 @@ func DefaultTestConfiguration(t ginkgo.GinkgoTInterface) {
...
 	} else {
-		log.Info().Msg("Env variable " + key + " not set")
+		log.Info().Msg(envVarLiteral + key + " not set")
```
Comment (shlomitk1) [inline manager/controllers/utils/configuration.go:101]: log.Info().Msgf("%s is undefined",key)

File: manager/controllers/app/fybrikapplication_controller.go [hunk 19]
```diff
@@ -69,16 +69,17 @@ type ApplicationContext struct {
 }
 
 const (
-	ApplicationTaxonomy = "/tmp/taxonomy/fybrik_application.json"
-	DataCatalogTaxonomy = "/tmp/taxonomy/datacatalog.json#/definitions/GetAssetResponse"
+	ApplicationTaxonomy   = "/tmp/taxonomy/fybrik_application.json"
+	DataCatalogTaxonomy   = "/tmp/taxonomy/datacatalog.json#/definitions/GetAssetResponse"
+	FybrikApplicationKind = "fybrikapplication"
```
Comment (roytman) [inline manager/controllers/app/fybrikapplication_controller.go:74]: I think it should be 	FybrikApplicationKind = "fybrikApplication"

**Review by roytman — COMMENTED** [review]
A nit comment

**Pull request merged**